src/aap_eda/core/management/commands/scheduler.py

```diff
@@ -0,0 +1,145 @@
+
+Example::
+
+  RQ_STARTUP_JOBS = [
+      {
+          "func": "package.module.funcname",
+          "scheduled_time": datetime(2020, 1, 1),
+      }
+  ]
+
+
+Periodic jobs.
+
+Example::
+
+    RQ_PERIODIC_JOBS = [
+        {
+            # Function to be queued
+            "func": "package.module.funcname",
+            # Time before the function is called again (in seconds).
+            "interval": 60,
+            # Time for first execution, in UTC timezone
+            ...
+        }
+    ]
+
+For a complete list of parameters refer to the
+https://github.com/rq/rq-scheduler/blob/master/README.rst
+
+Cron jobs.
+
+Example::
+
+    RQ_CRON_JOBS = [
+        {
+            # Function to be queued
+            "func": "package.module.funcname",
+            # A cron string
+            "cron_string": "0 0 * * 0",
+        }
+    ]
+
+For a complete list of parameters refer to the
+https://github.com/rq/rq-scheduler/blob/master/README.rst
+"""
+import logging
+from datetime import datetime
+from typing import NoReturn
+
+import django_rq
+from django.conf import settings
+from django_rq.management.commands import rqscheduler
+from rq_scheduler import Scheduler
+
+logger = logging.getLogger(__name__)
+
+
+RQ_STARTUP_JOBS = getattr(settings, "RQ_STARTUP_JOBS", None)
+RQ_PERIODIC_JOBS = getattr(settings, "RQ_PERIODIC_JOBS", None)
+RQ_CRON_JOBS = getattr(settings, "RQ_CRON_JOBS", None)
+
+
+def delete_scheduled_jobs(scheduler: Scheduler):
+    """Cancel any existing jobs in the scheduler when the app starts up."""
+    for job in scheduler.get_jobs():
+        logging.info("Deleting scheduled job: %s", job)
+        job.delete()
+
+
+def add_startup_jobs(scheduler: Scheduler) -> None:
+    if not RQ_STARTUP_JOBS:
+        logger.info("No scheduled jobs. Skipping.")
+        return
+
+    for entry in RQ_STARTUP_JOBS:
+        logger.info('Adding startup job "%s"', entry["func"])
+        scheduled_time = entry.pop("scheduled_time", None)
+        if scheduled_time is None:
+            scheduled_time = datetime.utcnow()
+        scheduler.enqueue_at(
+            scheduled_time=scheduled_time,
+            **entry,
+        )
+
+
+def add_periodic_jobs(scheduler: Scheduler) -> None:
+    if not RQ_PERIODIC_JOBS:
+        logger.info("No periodic jobs. Skipping.")
+        return
+
+    for entry in RQ_PERIODIC_JOBS:
+        logger.info('Adding periodic job "%s"', entry["func"])
+        scheduled_time = entry.pop("scheduled_time", None)
+        if scheduled_time is None:
+            scheduled_time = datetime.utcnow()
+        scheduler.schedule(
+            scheduled_time=scheduled_time,
+            **entry,
+        )
+
+
+def add_cron_jobs(scheduler: Scheduler) -> None:
+    if not RQ_CRON_JOBS:
+        logger.info("No scheduled jobs. Skipping.")
+        return
+
+    for entry in RQ_CRON_JOBS:
+        logger.info('Adding cron job "%s"', entry["func"])
+        scheduler.cron(**entry)
+
+
+class Command(rqscheduler.Command):
```

Comment:
We also need a `help` message for the command

Reply:
Done!